packages/core/carbon.yml

```diff
@@ -0,0 +1,527 @@
+      - part-to-whole
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/circle-pack.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#circle-packs
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/complex-charts-circle-pack
+  combo:
+    name: combo
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+    framework: vanilla
+    platform: web
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/getting-started
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-combo
+  donut:
+    name: Donut
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+      - part-to-whole
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/donut.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#donut
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-donut
+  gauge:
+    name: Gauge
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+      - part-to-whole
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/gauge.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts/#gauge
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-gauge
+  geo-choropleth:
+    name: Choropleth map
+    status: draft
+    type: component
+    tags:
+      - data-visualization
+      - geographic-overlay
+    framework: design-only
+    platform: web
+    thumbnailPath: /src/thumbnails/geo-choropleth.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#geographic-overlays
+  geo-connection:
+    name: Connecting lines
+    status: draft
+    type: component
+    tags:
+      - data-visualization
+      - geographic-overlay
+    framework: design-only
+    platform: web
+    thumbnailPath: /src/thumbnails/geo-connection.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#connection-map
+  geo-proportion:
+    name: Proportional symbol
+    status: draft
+    type: component
+    tags:
+      - data-visualization
+      - geographic-overlay
+    framework: design-only
+    platform: web
+    thumbnailPath: /src/thumbnails/geo-choropleth.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#proportional-symbol-map
+  heatmap:
+    name: Heat map
+    status:
+      key: experimental
+      note:
```

Comment:
@theiliad is Heat map experimental? I saw this note in Storybook.

Reply:
Yes we're still making adjustments to it

Reply:
Is this experimental for all? Or just vanilla and stable for others?